### PR TITLE
Add file ID to Asset Edit View in Upload plugin

### PR DESCRIPTION
**Pull request by derrickmehaffy**
## What this PR does

- [x] Adds the `File ID` to the edit view dialog in the upload plugin

## How to test

- Upload a file
- Check right side in the file attributes to see `File ID` and the internal integer ID

### CODE REVIEW

File: packages/core/upload/admin/src/components/EditAssetDialog/index.js [hunk 1]
```diff
@@ -225,6 +225,14 @@ export const EditAssetDialog = ({
                           }),
                           value: getFileExtension(asset.ext),
                         },
+
+                        {
+                          label: formatMessage({
+                            id: getTrad('modal.file-details.id'),
+                            defaultMessage: 'File ID',
```
Comment (gu-stav) [inline packages/core/upload/admin/src/components/EditAssetDialog/index.js:232]: In the admin app we call files `Assets`. Should we rename this to `Asset ID`?

**Comment by derrickmehaffy** [comment]
@gu-stav do we normally add in the translation keys by default with additions like this? I know the default value will set it if the key doesn't exist.

**Review by maevalienard — APPROVED** [review]

File: packages/core/upload/admin/src/components/EditAssetDialog/index.js (hunk 1 above)
Comment (gu-stav) [reply packages/core/upload/admin/src/components/EditAssetDialog/index.js]: Yeah, it is an inconsistency: the backend calls them files and the frontend assets ... 🤦🏼

**Comment by ronronscelestes** [comment]
> I think most times we add the new keys at least to the `en.json` file

Exactly 😊 `modal.file-details.id` translation key should be added in this file 
image

**Comment by derrickmehaffy** [comment]
> @derrickmehaffy I think we have different strategies. Asking @ronronscelestes here: I think most times we add the new keys at least to the `en.json` file, so that new developers who are creating new languages/ translations can copy from there ...
> 
> I think you also need to update the snapshot tests: `yarn test:front --updateSnapshot`

I'm honestly surprised we don't just build a GitHub action to do that :thinking: seems odd to have to run that manually for every PR.

**Comment by gu-stav** [comment]
It's not on every PR: only when you modify markup that is being snapshot tested. Currently, this acts as a safety-net, to understand the implications of a change (e.g. to see if something is not rendered any longer at a certain place). There is a lot of discussion going on about how useful they are - right now big parts of the app are only tested through these.

**Review by gu-stav — APPROVED** [review]
@derrickmehaffy I've fixed the snapshot tests for you (I think you might have been on an older version of the design-system and therefore the output didn't match).

The code looks good to me now and I think we can merge this. Thanks again 🌻

**Pull request merged**